### ISSUE TITLE
Segmentation fault on Ubuntu jammy with Qt6, "Qt WebEngine resources not found"

### ISSUE THREAD

**Issue by tsteven4** [gpsbabel/gpsbabel#931]
The following error occurs with gpsbabelfe when compiled with qt6-base-dev (6.2.4+dfsg-2ubuntu1).  This error is generated if "Preview in Google Maps"  is selected under "More Options".

This is due to a build issue with qt6-base-dev (6.2.4+dfsg-2ubuntu1) on jammy as discussed at https://bugs.launchpad.net/ubuntu/+source/qt6-base/+bug/1970057

Another result of this issue is that some translations cannot be found, specifically the ones provided by Qt.

Until ubuntu updates qt6-base-dev on jammy you may work around this issue by enabling a PPA that contains the fix for this issue.  Add the PPA to your system as instructed at:
https://launchpad.net/~tsteven4/+archive/ubuntu/qt6-backports
and then 
sudo apt-get update
sudo apt-get upgrade
This will install qt6-base-dev (6.2.4+dfsg-8~ubuntu22.04.1), which was once published for kinetic.  The PPA just builds the same source for jammy.

```
Qt WebEngine resources not found at /share/qt6/resources. Trying parent directory...
Qt WebEngine resources not found at /share/qt6. Trying application directory...
Qt WebEngine resources not found at /home/tsteven4/work/gpsbabel/bld/gui/GPSBabelFE. Trying fallback directory... The application MAY NOT work.
Path override failed for key base::DIR_QT_LIBRARY_DATA and path '/home/tsteven4/.GPSBabel'
Installed Qt WebEngine locales directory not found at location /share/qt6/translations/qtwebengine_locales. Trying application directory...
Qt WebEngine locales directory not found at location /home/tsteven4/work/gpsbabel/bld/gui/GPSBabelFE/qtwebengine_locales. Trying fallback directory... Translations MAY NOT not be correct.
Path override failed for key ui::DIR_LOCALES and path '/home/tsteven4/.GPSBabel'
[1011/072627.581618:ERROR:resource_bundle.cc(911)] Failed to load /home/tsteven4/.GPSBabel/qtwebengine_resources_100p.pak
Some features may not be available.
[1011/072627.581664:ERROR:resource_bundle.cc(911)] Failed to load /home/tsteven4/.GPSBabel/qtwebengine_resources_200p.pak
Some features may not be available.
[1011/072627.581681:ERROR:resource_bundle.cc(911)] Failed to load /home/tsteven4/.GPSBabel/qtwebengine_resources.pak
Some features may not be available.
[1011/072627.581703:WARNING:resource_bundle_qt.cpp(119)] locale_file_path.empty() for locale
[3369:3369:1011/072627.614852:ERROR:extension_system_qt.cpp(121)] Failed to parse extension manifest.
Segmentation fault
```

**Comment by tsteven4** [comment]
Ubuntu has released qt6-base-dev 6.2.4+dfsg-2ubuntu1.1 on jammy-updates with the fix.  If you installed the PPA above you may need to uninstall it and the related packages before picking up the updated ubuntu packages.

**Comment by tsteven4** [comment]
closed as ubuntu has released a fixed package.
